src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { Todo } from '../../types';
+import { UserInfo } from '../UserInfo/UserInfo';
+import { UserInfoAbsent } from '../UserInfo/UserInfoAbsent';
+
+interface Props {
+  todoItem: Todo;
+}
+
+export const TodoInfo: React.FC<Props> = ({ todoItem }) => {
+  const {
+    title, completed, foundUser, id,
+  } = todoItem;
+
+  return (
+    <li className={`todo-list__item ${completed === true ? 'todo-list__item--completed' : ''}`}>
```

Comment:
```suggestion
    <li className={`todo-list__item ${completed ? 'todo-list__item--completed' : ''}`}>
```

і цей вираз винеси в змінну треба `completed ? 'todo-list__item--completed' : ''`